src/client/App.js

```diff
@@ -53,15 +52,14 @@ function App() {
         </Route>
 
         {/* Anonymous pages */}
-        <SignIn exact path="/sign-in" />
-        <SignUp exact path="/sign-up" />
+        <CreateAccountPage exact path="/sign-up" />
         <ResetPassword exact path="/reset-password" />
```

Comment:
If you deleted the container for ResetPassword you need to remove this line also.

Reply:
I didn't delete ResetPassword because it has been used in several file.